src/ray/common/ray_config_def.h

```diff
@@ -330,9 +330,17 @@ RAY_CONFIG(int64_t, max_placement_group_load_report_size, 100)
 /// Python IO workers to determine how to store/restore an object to/from
 /// external storage.
 RAY_CONFIG(std::string, object_spilling_config, "")
+
 /// Whether to enable automatic object spilling. If enabled, then
 /// Ray will choose objects to spill when the object store is out of
 /// memory.
 RAY_CONFIG(bool, automatic_object_spilling_enabled, true)
+
 /// The maximum number of I/O worker that raylet starts.
 RAY_CONFIG(int, max_io_workers, 1)
+
+/// The minimum size that can be spilled. 100 MB by default.
+/// This value is recommended to be smaller than the plasma object size, but
+/// it will still work although it doesn't.
+/// Setting 0 for this config means that you don't fusion objects.
```

Comment:
```suggestion
/// Setting 0 for this config means that small objects won't be fused.
```

Reply:
Actually is this comment accurate? It looks like if enough space is needed (because someone is trying to create a large object), it will still fuse the spilled objects even if this is set to 0. Also, if the total spillable object size is less than this value, it seems like it will still spill those objects immediately. Is that right?

Reply:
Thanks for catching this! You are definitely right. This can fusion objects if the num_bytes require multiple objects to dump into a single spill worker.